roles/gerrit/templates/ssh.config.j2

```diff
@@ -0,0 +1,5 @@
+{% for repo in gerrit_replication %}
+Host {{repo.repo}}.github.com
+    Hostname github.com
+    IdentityFile ~/.ssh/{{repo.keyfile}}
```

Comment:
Would it be more future proof to prefix the key with github- ? (in case we later have to push to gitlab, etc)